designs/FloatingVersions-In-PMUI.md

```diff
@@ -0,0 +1,103 @@
+
+# Floating Versions in the Package Manager UI
+
+* Status: In Review
+* Author(s): [Martin Ruiz](https://github.com/martinrrm)
+* Issue: [3788](https://github.com/NuGet/Home/issues/3788) PackageReference with a star in version is displayed under the updates tab/incorrectly represented in the installed tab
+
+## Problem Background
+
+Currently the user can specify a version range or a floating version, but the current behavior of the package manager UI has poor support for this, displaying an incorrect installed version. If the user indicates a floating version in any package (eg `5.*`) the PM UI will display the minimum version possible (`5.0.0`) as installed even though the resolved version could be different of the resolved version in the command `dotnet list package`. Furthermore the user cannot specify a version range free hand.
```

Comment:
Change from: `resolved version could be different of the`  `to resolved version could be different from the `
A couple of small suggestions:

```suggestion
In PackageReference the user can specify a version range or a floating version, but the package manager UI has poor support for this, displaying an incorrect installed version. 
If the user indicates a floating version for any package (eg `5.*`) the PM UI will display the minimum version possible (`5.0.0`) as installed even though the resolved version could be different of the resolved version in the command `dotnet list package`. 
Furthermore the user cannot specify a version range free hand.
```